interface/main/finder/dynamic_finder_ajax.php

```diff
@@ -83,10 +85,12 @@
     $where .= $where ? ' AND' : 'WHERE';
     $sSearch = add_escape_custom($_GET["sSearch_$i"]);
     if ($colname == 'name') {
-      $where .= " ( " .
-        "lname LIKE '$sSearch%' OR " .
-        "fname LIKE '$sSearch%' OR " .
-        "mname LIKE '$sSearch%' )";
+      //$where .= " ( " .
+      //  "lname LIKE '$sSearch%' OR " .
+      //  "fname LIKE '$sSearch%' OR " .
+      //  "mname LIKE '$sSearch%' )";
+	   $where .= " ( " .
+		"CONCAT(lname,' ',fname,' ',mname) LIKE '%$sSearch%')";
```

Comment:
You are replacing a compound wild search by single simple wild search.  In those cases there are inherent issues of false positives.  This should be tested for its implications or use special characters.